extra-security/clipass/spec

```diff
@@ -0,0 +1,5 @@
+VER=0.2.5
+# REL=0 The package revision. If not specified, it's 0.
+SRCTBL="https://github.com/neruthes/Clipass/archive/v$VER.tar.gz"
+DUMMYSRC=1
```

Comment:
Why are you using `$DUMMYSRC` and `$SRCTBL` at the same time? It looks like you're not using that tarball.

Reply:
Fixed.

Reply:
Fixed.